features/Extended Materials/Shaders/ExtendedMaterials/ExtendedMaterials.hlsli

```diff
@@ -14,14 +14,18 @@ struct DisplacementParams
 
 namespace ExtendedMaterials
 {
-	float AdjustDisplacement(float displacement, DisplacementParams params)
+	float DisplacementHeight0(float displacement, DisplacementParams params)
```

Comment:
`DisplacementHeight0` and `Displacement01` are really bad naming

Reply:
Does ScaledDisplacement and NormalizedDisplacement sound better? or something like that

Reply:
Certainly better